randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -88,7 +88,9 @@ Extra - asset_id: collision_camera_001
   > Grapple Block Alcove
       All of the following:
           Morph Ball and After Ghavoran - Burenia Transport Grapple Box
-          Grapple Beam or Spider Magnet or Simple IBJ or Use Spin Boost
+          Any of the following:
+              Spider Magnet or Speed Booster or Simple IBJ or Use Spin Boost
+              Grapple Beam and Movement (Beginner)
```

Comment:
I'm not sure if this grapple beam requirement should require beginner movement - there's a similar requirement done in these two spots with no movement at all:

- Artaria - Energy Recharge Station South, going from Energy Recharge -> Missile Tank
- Dairon - Hub Access, going from Above Crumble Blocks -> Dock to Save Station East

At the very least, they should be consistent with each other?

Reply:
Yeah, I've been trying to make ones that you have to use the grapple momentum or jump out of it as a movement trick. forgot to do the one in Artaria i guess.